packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -539,7 +540,10 @@ const AnalyticalTable = forwardRef((props: AnalyticalTablePropTypes, ref: Ref<HT
     return props.data;
   }, [props.data, minRows]);
 
-  const tableInstanceRef = useConsolidatedRef<Record<string, any>>(tableInstance);
+  let tableInstanceRef = useRef<Record<string, any>>();
+  if (tableInstance) {
+    tableInstanceRef = tableInstance;
+  }
```

Comment:
This is technically a breaking change - we have accepted a callback ref for the `tableInstance` before that change